ERCS/erc-7549.md

```diff
@@ -0,0 +1,111 @@
+---
+title: <Inscription In Smart Contract>
```

Comment:
```suggestion
title: Inscription In Smart Contract
```

Reply:
All angle brackets <> in the preamble have been removed.